README.md

```
@@ -265,14 +283,17 @@ make build
 - **M** — quickly mute/unmute the audio stream while the main window is focused
 
 ### Default Signal Chain
+
 The application starts with a clean acoustic preset. Only EQ and Reverb are enabled by default — all other effects start bypassed:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)